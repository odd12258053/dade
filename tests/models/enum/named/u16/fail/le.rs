use dade::model;
#[model]
enum TestModel {
    Value {
        #[field(le = 2.0)]
        value: u16
    },
}
fn main() {}
