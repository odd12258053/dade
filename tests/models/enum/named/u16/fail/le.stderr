error[E0277]: can't compare `u16` with `{float}`
 --> tests/models/enum/named/u16/fail/le.rs:2:1
  |
2 | #[model]
  | ^^^^^^^^ no implementation for `u16 < {float}` and `u16 > {float}`
  |
  = help: the trait `PartialOrd<{float}>` is not implemented for `u16`
  = help: the following other types implement trait `PartialOrd<Rhs>`:
            f32
            f64
            i128
            i16
            i32
            i64
            i8
            isize
          and $N others
  = note: this error originates in the attribute macro `model` (in Nightly builds, run with -Z macro-backtrace for more info)
